### ISSUE TITLE
Hyper-params to reproduce results

### ISSUE THREAD

**Issue by abhinavkashyap** [CuthbertCai/pytorch_DAN#4]
Hi, 

Thank you for the repository. What hyper-parameters did you set for your reported results? How many epochs and what is the learning rate :) Please let us know 

Thank you 
Best, 
Abhinav

**Comment by CuthbertCai** [comment]
I used the default setting as shown in `params.py`. The result reported in this repo was the best one in 100 epochs (just for a demo, instead of a published paper).

**Comment by abhinavkashyap** [comment]
Thank you :) I will try running the code.. Cheers!
